deposit-to-exchange.md

```diff
@@ -1 +1,138 @@
-# Title
+# Confirm a Deposit to an Exchange
+
+Deposits are made in several stages. 
+One of the stages is to go through the [Exchange server](https://docs.flare.network/exchange/architecture/#architecture-of-an-exchange), which monitors for deposits and updates the Balances database. 
+Run the code below to have the Exchange server listen for deposit transactions and confirm the deposits.
+
+On a blockchain, multiple [validators](h[ttps://docs.flare.network/tech/glossary/](https://docs.flare.network/tech/validators/)) examine newly created transactions and come to an agreement about whether they are valid. 
+When they are valid, they are added to a new block on the chain and identified by a one-way cryptographic algorithm, called a _hash_. 
+The one-way hash prevents transactions from being reverted. 
+[Blocks](https://docs.flare.network/tech/glossary/) are added to the [blockchain](https://docs.flare.network/tech/glossary/) in chronological order and it may take a few blocks for the hash to be completed.
+To prevent problems, we confirm deposits several blocks back (five blocks in our code), when the blocks are most likely to be irreversible.
+
```

Comment:
The [Architecture of an Exchange doc](https://docs.flare.network/exchange/architecture/) says, "To avoid problems, the Exchange should only act on transactions appearing on blocks old enough for the chance of them being reverted to be negligible."  Is this doc referring to the fact that blocks are irreversible except in the case of a severe attack when an ecosystem might agree to fork. Am I on the right track? 
 I have never heard of a period where they could possibly be reverted, so I'm imagining that is the brief moment waiting for the cryptographic hash to be in place, yes?